person.rb

```diff
@@ -0,0 +1,29 @@
+class Person
+  attr_accessor :name
+  attr_reader :id, :age
+
```

Comment:
- [ ] Good job creating the Person class , 👏 . As for this project requirements , the `age` should be able to be modified , and read. Kindly define it as `attr_accessor` to meet the requirement 👍